### PR TITLE
[CTCP-5144] add additional information code page

### PR DESCRIPTION
**Pull request by prashanthmamidi**

### CODE REVIEW

File: app/controllers/additionalInformation/index/AdditionalInformationTypeController.scala [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        val updatedList = SelectableList.apply(
+          additionalInformationList.values.filter(
+            code => !code.value.equals("30600")
+          )
+        )
+        (formProvider("additionalInformation.index.additionalInformationType", updatedList), updatedList)
+
+      case _ =>
+        (formProvider("additionalInformation.index.additionalInformationType", additionalInformationList), additionalInformationList)
+    }
```
Comment (kenneth-rayner) [inline app/controllers/additionalInformation/index/AdditionalInformationTypeController.scala:78]: Could move this def to above the onPageLoad method and also refactor  to use filterNot, eg:
 private def getFormAndUpdatedCodeList(request: DataRequest[AnyContent], additionalInformationList: SelectableList[AdditionalInformationCode]) =
    request.userAnswers.get(ItemsDestinationCountryInCL009Page) match {
      case Some(true) =>
        val updatedList = SelectableList.apply(additionalInformationList.values.filterNot(_.value.equals("30600")))
        (formProvider("additionalInformation.index.additionalInformationType", updatedList), updatedList)
      case _ => (formProvider("additionalInformation.index.additionalInformationType", additionalInformationList), additionalInformationList)
    }

**Comment by kenneth-rayner** [comment]
@prashanthmamidi  Can you please add screenshots?

**Comment by prashanthmamidi** [comment]
image

**Comment by prashanthmamidi** [comment]
image

**Review by kenneth-rayner — APPROVED** [review]

File: app/controllers/additionalInformation/index/AdditionalInformationTypeController.scala (hunk 1 above)
Comment (kenneth-rayner) [reply app/controllers/additionalInformation/index/AdditionalInformationTypeController.scala]: @prashanthmamidi  Move to service layer?

**Pull request merged**